tools/GSDumpGUI/Library/GSDXWrapper.cs

```diff
@@ -39,14 +39,15 @@ namespace GSDumpGUI
     public delegate void GSreset();
     public delegate void GSreadFIFO2(IntPtr data, int size);
     public delegate void GSsetGameCRC(int crc, int options);
-    public delegate int GSfreeze(int mode, IntPtr data);
-    public delegate void GSopen(IntPtr hwnd, String Title, int renderer);
+    public delegate  int GSfreeze(int mode, IntPtr data);
+    public delegate  int GSopen(IntPtr hwnd, String Title, int renderer);
```

Comment:
Shouldn’t you then align on the longest type? :D

Reply:
Is the new alignment better?